### ISSUE TITLE
ParentCallSpacing should consider starting or ending placement within class method.

### ISSUE THREAD

**Issue by mikebronner** [slevomat/coding-standard#1592]
Consider the following example:

```php
<?php

declare(strict_types=1);

namespace ValmarCodingStandard\Tests\Fixtures\Classes;

class Foo extends Bar
{
    public function __construct()
    {
        parent::__construct();
        parent::test();
    }

    public function test(): void
    {
        echo "test";

        parent::__construct();
        parent::test();

        echo "test";
}

```

```xml
	<rule ref="SlevomatCodingStandard.Classes.ParentCallSpacing">
        <properties>
            <property name="linesCountBefore" type="integer" value="0" />
            <property name="linesCountBeforeFirst" type="integer" value="1" />
            <property name="linesCountAfter" type="integer" value="0" />
            <property name="linesCountAfterLast" type="integer" value="1" />
        </properties>
    </rule>
```

There doesn't seem to be a way to allow for the parent grouping to be at the start or end of the method with no empty line before or after the group, while requiring the parent group to have an empty line when in between other code.
